drftest/tests/test_views.py

```diff
@@ -11,14 +11,22 @@
 
 
 class DummyView(ViewSet):
```

Comment:
Would you please add a very basic endpoint that generates excel and write some minimal tests test for it?

Reply:
Done 498d8a6